bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -158,7 +161,33 @@ f_dest.attrs['fit_param'] = args.fit_param
 f_dest.attrs['log_param'] = args.log_param
 with h5py.File(args.template_file, "r") as f_src:
     f_src.copy(f_src["./"], f_dest["./"], "input_template_params")
-temp_samples = numpy.vstack((template_pars)).T
+
+
+# Rejection Sampling
```

Comment:
This can only be done if 'mchirp' is one of the fit params. So I would add a check here to produce an error if the fit params list does not include mchirp

Reply:
I also suggest adding an attribute to the output file to record the value of the power, ie `attrs['mchirp_downsample_power']`